### PR TITLE
Parthj unit test time zone api controller.js

### PR DESCRIPTION
**Pull request by Parth-tech**
# Description
This pull request consists of unit tests for `timeZoneAPIController.js`

## Related PRS (if any):
None

## Main changes explained:
- Created 2 `.md` files highlighting the test cases to be covered (names of the .md files: `getTimeZoneProfileInitialSetup.md` and `getTimeZone.md`)
- Created a `..../controllers/timeZoneAPIController.spec.js` file containing all the unit tests.

## How to test:
1. check into current branch (i.e. `git checkout parthj_unit_test_timeZoneAPIController.js`)
2. do `npm install` 
3. do `npm run test timeZoneAPIController`

## Screenshots or videos of changes:
After successful execution it might look as follows:
Screenshot 2024-07-12 at 10 08 27 PM

### CODE REVIEW

**Comment by ZhangPeizhou** [comment]
Worked as expected, and passed all the tests. Well done!
PR#1027

**Review by aaryaneil — APPROVED** [review]
Tested the PR and all the test cases passed.
Screenshot 2024-07-13 at 2 54 36 AM

**Review by sahithidhulipala9 — APPROVED** [review]
All test cases passed!
PR 1027

**Review by nyelmela — APPROVED** [review]
I have tested this PR and all test cases have passed.
image

**Review by Rishitha1309 — APPROVED** [review]
All test cases passed.

**Review by jinxyou — APPROVED** [review]
All test cases have passed. Great work!
Screenshot 2024-07-13 160541

**Review by nikhilpittala16 — APPROVED** [review]
I have tested the PR and all the test cases have passed.
Screenshot (91)

**Review by linkhanthtel — APPROVED** [review]
All test cases passed!

PR #1027

**Review by Shadhrush5 — APPROVED** [review]
Hi, I tested PR #1027 as instructed and all the test cases are passing. Thank you for the great work.

Screenshot 2024-07-18 at 11 05 50 AM

**Comment by nikethaanand** [comment]
All the test cases are passed and working as expected.
image

**Review by ishan-miglani — APPROVED** [review]
I have tested the changes and all the test cases pass. Nice work!
Screenshot 2024-07-18 at 8 43 32 PM

File: requirements/timeZoneAPIController/getTImeZone.md
Comment (kabraambika) [inline requirements/timeZoneAPIController/getTImeZone.md:1]: Please fix the file name to `getTimeZone.md`. it is currently named `getTImeZone.md`.

File: package-lock.json
Comment (kabraambika) [inline package-lock.json:1]: If there are no changes in package.json, there's no need to submit package-lock.json.

**Comment by kabraambika** [comment]
I have tested the changes and all test cases are passing.
PR1027

**Review by kabraambika — CHANGES_REQUESTED** [review]
There are two comments provided to fix.

**Review by SaiLavanya1 — APPROVED** [review]
I have tested this PR, it is worked as expected.
Screenshot 2024-07-19 at 2 01 15 PM

**Review by jinxyou — APPROVED** [review]
All test cases have passed. Great work!
image

**Comment by cecilia-uu** [comment]
截屏2024-07-21 下午3 12 36
Tests have passed but the merging process may have some issue.

**Review by gowthamdongari — APPROVED** [review]
test cases works as expected
image

**Review by angelalalacheng — APPROVED** [review]
Hi, 
I use your branch `parthj_unit_test_timeZoneAPIController.js` to test and all pass.

Screenshot 2024-07-21 at 4 34 03 PM

**Review by Chetan0606 — APPROVED** [review]
Everything looks good
image

**Review by SrikanthChowdary495 — APPROVED** [review]
I have tested this PR, and passed all the tests. I have reviewed the code. 
test

**Review by ReinaT5678 — APPROVED** [review]
I ran the test and they all passed!

Screenshot 2024-07-24 at 9 05 39 PM

**Review by Annezhang673 — APPROVED** [review]
All tests cases passed. Works as expected.
PR1027

**Pull request merged**